pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -33,6 +33,7 @@ spec:
             - "--leader-election-lease-duration=137s"
             - "--leader-election-renew-deadline=107s"
             - "--leader-election-retry-period=26s"
+            - "--default-fstype=ext4"
```

Comment:
This would be overridden by the fstype in the storage class?

Reply:
previously the attacher sidecar always started with `ext4` as the default value and from v4.0.0 onwards the default is `""/nil.` https://github.com/kubernetes-csi/external-attacher/pull/358/files#diff-f422861635f15f707a5f60c81e05a96a541169e64b5f2cae6e0e964f7181b640R69 . Hence the CSI drivers that depended on `ext4` as the default need to change their deployments to explicitly set `--default-fstype=ext4`. The SC fstype setting will be respected by the provisioner and the set value will be send by the provisioner as part of the provision request.